clouddq/main.py

```diff
@@ -406,15 +404,40 @@ def main(  # noqa: C901
         metadata = json.loads(metadata)
         # Load Rule Bindings
         configs_path = Path(rule_binding_config_path)
-        logger.debug("Loading rule bindings from: %s", configs_path.absolute())
+        logger.debug("Loading rule bindings from: {configs_path.absolute()}")
         all_rule_bindings = lib.load_rule_bindings_config(Path(configs_path))
         # Prepare list of Rule Bindings in-scope for run
         target_rule_binding_ids = [r.strip() for r in rule_binding_ids.split(",")]
         if len(target_rule_binding_ids) == 1 and target_rule_binding_ids[0] == "ALL":
             target_rule_binding_ids = list(all_rule_bindings.keys())
+        logger.debug(f"Preparing SQL for rule bindings: {target_rule_binding_ids}")
+        # Load default configs for metadata registries
+        registry_defaults: MetadataRegistryDefaults = lib.load_metadata_registry_default_configs(Path(configs_path))
```

Comment:
add test for loading multiple metadata_registry_defaults fail: - multiple metadata_registry_defaults with different values, - loading metadata_registry_defaults with bigquery key fails

Reply:
there should be a new fixture for metadata_registry_defaults in conftest.py, loaded using lib.load_metadata_registry_default_configs(Path(configs_path))

Reply:
each of the above tests should be in test_metadata_registry_defaults using different input configs from resources

Reply:
todo: thinh